app/controllers/like_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikeController < ApplicationController
```

Comment:
- [ ] Kindly rename this file to `likes_controller.rb`.

- [ ] Kindly rename this class to `LikesController`, instead of `LikeController`. In Rails, controllers use the plural form of names.